project3/index.html

```diff
@@ -0,0 +1,409 @@
+ <!DOCTYPE html>
+ <html lang="en">
+
+
+ <head>
+     <meta charset="UTF-8">
+     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+     <link rel="stylesheet" href="style.css">
+     <title>Document</title>
```

Comment:
Your `title` element should be changed to something specific to your page/project/site. This is the default text.